src/loader/mod.rs

```diff
@@ -67,6 +63,10 @@ pub enum Error {
     InvalidEntryAddress,
     /// Invalid bzImage binary.
     InvalidBzImage,
+    /// Invalid Image binary.
+    InvalidImage,
```

Comment:
The newly added error types are `aarch64`-specific, please add appropriate `cfg(target)`s.

Reply:
done